src/odemis/odemisd/main.py

```diff
@@ -662,7 +662,7 @@ def main(args):
     # The settings file is opened here because root privileges are dropped at some point after
```

Comment:
These changes seem to be from a different commit in Odemis upstream. Probably rebasing against upstream would drop this from the PR, which would be handy, as to merge your PR, I'll do a squash, which would bring conflicts.